doc/source/whatsnew/v0.23.0.txt

```diff
@@ -890,6 +890,7 @@ Groupby/Resample/Rolling
 - Bug in :func:`DataFrame.groupby` passing the `on=` kwarg, and subsequently using ``.apply()`` (:issue:`17813`)
 - Bug in :func:`DataFrame.resample().aggregate` not raising a ``KeyError`` when aggregating a non-existent column (:issue:`16766`, :issue:`19566`)
 - Fixed a performance regression for ``GroupBy.nth`` and ``GroupBy.last`` with some object columns (:issue:`19283`)
+- Bug in :func:`DataFrame.groupby.cumsum` and :func:`DataFrame.groupby.cumprod` where skipna is not an option
```

Comment:
These should be

    :meth:`DataFrameGroupBy.cumsum`

likewise for cumprod.

And skipna should probably be in double backticks.

And could you add a `:issue:` for the issue you're closing.

Reply:
Added those in